ansible/content-servers.yml

```diff
@@ -0,0 +1,197 @@
+- hosts: bananas_fileservers
+  gather_facts: true
+  become: true
+
+  vars:
+    # Our vars, for easy reuse
+    cache_timeout: "1y"  # Basically just as long as nginx runs for
+    bananas_server: bananas.openttd.org
+    bananas_cdn_server: bananas.cdn.openttd.org
```

Comment:
As mentioned on IRC, we also need a `bananas.cdn.staging.openttd.org` variant, so we can test this before hitting production